### DISCUSSION TITLE
Scoop installation command installs to D:\Applications rather than user directory

### DISCUSSION THREAD

**Discussion by Camuise** [ScoopInstaller/Scoop#5007] [Help and Questions]
When trying to install scoop using `irm get.scoop.sh | iex` the installer installs scoop to D:\Applications and refuses to install anything if the D: drive is not connected, throwing the following error:
```New-Item:
Line |
 490 |          New-Item -Type Directory $SCOOP_APP_DIR | Out-Null
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Could not find a part of the path 'D:\Applications\Scoop\apps\scoop\current'.```

**Comment by rashil2000** [comment]
Paste the output of `gci env:`

**Comment by Camuise** [comment]
> Paste the output of `gci env:`

Where would I find this

**Comment by rashil2000** [comment]
It's a command, you have to run this in PowerShell
